src/llmcompressor/transformers/utils/helpers.py

```diff
@@ -54,3 +59,223 @@ def detect_last_checkpoint(
             )
 
     return last_checkpoint
+
+
+def is_model_quantized_from_path(path: str):
```

Comment:
this seems redundant with the method we have in text_generation?

Reply:
sorry which method? 
is_model_quantized_from_path is defined here, text_generation.py uses it from here